manage.py

```diff
@@ -29,42 +29,65 @@ def seed_categories():
 
     food = Category.get_by_name("Food")
     if not food:
-        food = Category("Food", "", 0)
+        food = Category()
+        food.name = "Food",
+        food.description = ""
+        food.sort_order = 0
```

Comment:
Why was this changed? If the model's init needs to be updated to support the right parameters, then it should be.